config/initializers/new_framework_defaults_5_2.rb

```diff
@@ -0,0 +1,35 @@
+# Be sure to restart your server when you modify this file.
```

Comment:
If we can't flip any of these now, let's make an issue for each.

Reply:
Created a ticket with a checklist: https://github.com/lobsters/lobsters/issues/509